packages/conf-llvm/conf-llvm.3.9/files/source-linux.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+sudo add-apt-repository 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.9 main'
+curl -L http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
```

Comment:
I don't know what the opam conventions are, but I am a bit uncomfortable at the idea that a script I am using without reading it first would add GPG keys to my apt trusted set (it does ask me for my sudo password in some cases, but with not enough information to judge on whether I want to give the permission). Would it be reasonable to, instead, check that the repository exists, and instead fails with a message giving these commands for the user to execute?

What is `"source" "linux"` as a depext predicate anyway? This is the first time I have seen it, when does it apply?